src/serverless.js

```
@@ -7,17 +7,39 @@ const fs = require("fs");
 const { createOrUpdateMetaRole } = require("./utils");
 
 class LambdaCron extends Component {
+  validate(inputs) {
+    if (!inputs.schedule) {
+      throw new Error(
+        "Input 'schedule' is required. Please see README: https://git.io/JJWW0"
+      );
+    }
+    const { schedule }=inputs.schedule
+
+    let valid = false
+
+    // Check for a cron expression
+    let pat = /(@(annually|yearly|monthly|weekly|daily|hourly|reboot))|(@every (\d+(ns|us|µs|ms|s|m|h))+)|((((\d+,)+\d+|(\d+(\/|-)\d+)|\d+|\*) ?){5,7})/
+    if (pat.test(schedule)) {
+      valid = true;
+    }
+
+    // Check for a rate expression
+    pat = /rate(\d+\s+(minute|minutes|hour|hours|day|days))/
+    if (pat.test(schedule)) {
+      valid = true
+    }
+
+    if(!valid) {
+      console.error("Schedule string is possibly wrong. Please recheck it.")
```

Let's throw an error here if the expression is invalid